content/tutorial/tutorial.md

```diff
@@ -12,132 +12,130 @@ redirect_from:
   - "docs/tutorial-zh-CN.html"
 ---
 
-This tutorial doesn't assume any existing React knowledge.
+Tutorial ini tidak mengasumsikan setiap pengetahuan tentang React yang sudah ada. 
 
-## Before We Start the Tutorial {#before-we-start-the-tutorial}
+## Sebelum Kita Memulai Tutorial {#before-we-start-the-tutorial}
 
-We will build a small game during this tutorial. **You might be tempted to skip it because you're not building games -- but give it a chance.** The techniques you'll learn in the tutorial are fundamental to building any React apps, and mastering it will give you a deep understanding of React.
+Kita akan membangun sebuah *game* kecil sepanjang tutorial ini. **Anda bisa jadi tergoda untuk melewati tutorial ini karena Anda membuat *game* bukan pekerjaan utama Anda -- Namun, luangkan waktu untuk mencobanya.** Teknik-teknik yang akan dipelajari pada tutorial ini adalah hal-hal dasar yang digunakan untuk membangun aplikasi React. Dengan menguasai teknik-teknik tersebut, Anda akan memehami React secara mendalam.
 
->Tip
+>Tips
 >
->This tutorial is designed for people who prefer to **learn by doing**. If you prefer learning concepts from the ground up, check out our [step-by-step guide](/docs/hello-world.html). You might find this tutorial and the guide complementary to each other.
+> Tutorial ini didesain untuk orang yang lebih suka **belajar dengan mempraktikan**. Jika Anda lebih suka belajar mempelajari konsep dari bawah ke atas, lihat [panduan langkah demi langkah](/docs/hello-world.html) kami. Anda akan menyadari bahwa tutorial dan panduan akan saling melengkapi.
 
-The tutorial is divided into several sections:
+Tutorial ini akan dibagi ke beberapa bagian:
 
-* [Setup for the Tutorial](#setup-for-the-tutorial) will give you **a starting point** to follow the tutorial.
-* [Overview](#overview) will teach you **the fundamentals** of React: components, props, and state.
-* [Completing the Game](#completing-the-game) will teach you **the most common techniques** in React development.
-* [Adding Time Travel](#adding-time-travel) will give you **a deeper insight** into the unique strengths of React.
+* [Mempersiapkan Tutorial](#setup-for-the-tutorial) merupakan **titik awal** Anda untuk mengikuti tutorial.
+* [Ikhtisar](#overview) bagian ini berisi **hal-hal mendasar** React: komponen, *props*, dan *state*.
+* [Menyelesaikan Permainan](#completing-the-game) bagian ini berisi **teknik-teknik yang paling umum** pada pengembangan aplikasi React.
+* [Menambahkan Penjelajahan Waktu](#adding-time-travel) bagian ini akan memberikan **wawasan yang mendalam** pada kekuatan unik dari React.
 
-You don't have to complete all of the sections at once to get the value out of this tutorial. Try to get as far as you can -- even if it's one or two sections.
+Anda tidak perlu menyelesaikan semua bagian sekaligus untuk mendapatkan pembelajaran dari tutorial ini. Cobalah mengerjakan semampu Anda walaupun hanya satu atau dua bagian.
```

Comment:
I think we should keep every punctuation as close as possible to the original docs, so:

Cobalah mengerjakan semampu anda -- walaupun hanya satu atau dua bagian.